src/controllers/publish.ts

```diff
@@ -533,28 +533,25 @@ export const measurePreview = async (req: Request, res: Response, next: NextFunc
         return;
       } catch (err) {
         const error = err as ApiException;
-        logger.debug(`Error is: ${JSON.stringify(error, null, 2)}`);
-        if (error.status === 400) {
+        const viewErr = JSON.parse((error.body as string) || '{}') as ViewErrDTO;
+        logger.debug(`Error is: ${JSON.stringify(viewErr, null, 2)}`);
+        if (viewErr.status === 400) {
```

Comment:
We shouldn't ever need to use the body for the http status code - the backend response status code is added as a property of ApiException (`error.status`).

Reply:
Switched to using `error.status`